src/components/DateRangePicker.js

```diff
@@ -274,7 +275,7 @@ class DateRangePicker extends React.Component {
     const shouldShowCalendarIcon = StyleUtils.getWindowSize(theme.BreakPoints) !== 'small';
 
     return (
-      <div style={styles.component}>
+      <div ref={this.props.dateRangePickerRef} style={styles.component}>
```

Comment:
Let's move this to the `a` on the next line because it has the `tabIndex` and key handling for this component.